notebooks/MAGICC7.ipynb

```diff
@@ -108,46 +132,151 @@
     "  FILE_TUNINGMODEL_8 =  \"USER\",\n",
     "  FILE_TUNINGMODEL_9 =  \"USER\",\n",
     "  FILE_TUNINGMODEL_10 =  \"USER\",\n",
+    "  # at least one `OUT_` variable must be set to 1 to produce output\n",
+    "  # for this notebook, since we only plot temperature, we only need OUT_TEMPERATURE\n",
+    "  OUT_TEMPERATURE = 1,\n",
     "```"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 4,
+   "execution_count": 5,
    "metadata": {},
    "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "['rcp26', 'rcp45', 'rcp60', 'rcp85']"
+      ]
+     },
+     "execution_count": 5,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# NBVAL_SKIP\n",
+    "scenario_names = list(map(str.lower, scenarios.rcps.meta[\"scenario\"].unique()))\n",
```

Comment:
`scenarios.items()` no longer exists so I've used this method to obtain all the available rcps.